network-api/networkapi/wagtailcustomization/static/css/admin.css

```diff
@@ -0,0 +1,7 @@
+/*
+  This is 16% by default, which is absolutely ridiculous.
+*/
+fieldset ul li div label {
+  display: block;
+  width: auto;
+}
```

Comment:
Can we move CSS code in this [file](https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/wagtailcustomization/templates/wagtailadmin/base.html) to here? If so, I will file a new ticket.

Reply:
yeah, that's a good idea!

Reply:
filed https://github.com/mozilla/foundation.mozilla.org/issues/5290